apiary.apib

```diff
@@ -541,6 +541,80 @@ API endpoints that Operation Code's Rails backend makes available to its React f
                 status: "ok"
             }
 
++ Response 422 (application/json)
+
+        {
+            errors: "Some error message"
+        }
+
+## Mentor | Collection [/api/v1/mentors/{?id,email,zip,first_name,last_name,slack_name,mentor,verified}]
```

Comment:
When you are passing parameters to a `GET`, you will not have a `/` that separates them from the main URL.   So you'll remove the `/` between `mentors` and `{`.